### PR TITLE
New site: Rough home draft

### PR DESCRIPTION
**Pull request by guimachiavelli**
Trying to sketch what the new home would look like.

- Remove News section
- Remove most text
- Replace SDK reference `Capsule` content with Quick start link
- Replace placeholder `Featured` content with references to API reference, SDK page, Meilisearch Cloud, and blog

Reference of more or less what I have in mind (ignore the icons)

### CODE REVIEW

File: home.mdx [hunk 1]
```diff
@@ -3,51 +3,35 @@ title: Meilisearch Documentation
 description: Here you'll find everything you need to learn and create with our lightning fast search engine.
 ---
 
-# Welcome to the Meilisearch documentation
+# Documentation
 
-Here you'll find everything you need to learn and create with our lightning fast search engine.
+Explore our guides and references to learn how to use Meilisearch in your projects.
```
Comment (dichotommy) [inline home.mdx:8]:
```suggestion
Learn to use Meilisearch in your projects by exploring our guides and API reference.
```

File: home.mdx [hunk 2]
```diff
@@ -3,51 +3,35 @@ title: Meilisearch Documentation
...
   {
-    content: 'Check what we will be working on in the next few weeks!',
+    content: 'Looking for SDK documentation? Check out this list of [official Meilisearch libraries](/learn/what_is_meilisearch/sdks.mdx).',
     icon: 'compass'
   },
   {
     content:
-      'Check out this list of official Meilisearch libraries or use the Integrations dropdown menu at the top of the screen for quick access.',
-    icon: 'user'
+      'Announcing [Meilisearch Cloud](https://cloud.meilisearch.com/register): the best way to add Meilisearch to your project',
```
Comment (dichotommy) [inline home.mdx:29]:
```suggestion
      '[Meilisearch Cloud](https://cloud.meilisearch.com/register): the easiest way to add Meilisearch to your project',
```

File: home.mdx [hunk 3]
```diff
@@ -3,51 +3,35 @@ title: Meilisearch Documentation
...
-    icon: 'circle'
-  },
-  {
-    icon: 'globe',
-    content: 'Check what we will be working on in the next few weeks!'
-  },
-  {
-    content: 'Check what we will be working on in the next few weeks!',
-    icon: 'letter'
+      'Read the Meilisearch blog for more tips, tutorials, and information about our company.',
```
Comment (dichotommy) [inline home.mdx:34]:
```suggestion
      'For more tips, tutorials, and information about our company, take a look at the Meilisearch blog.',
```

File: home.mdx [hunk 4]
```diff
@@ -3,51 +3,35 @@ title: Meilisearch Documentation
...
-<h2 className="text-5xl">Meilisearch news</h2>
-
-Announcing Meilisearch Cloud: [register now](https://cloud.meilisearch.com/register) or [log in here](https://cloud.meilisearch.com/login) if you already have an account.
+<Capsule>
+  ## Quick start
 
-<Divider />
-
-We just raised a 15M Series A! 🥳 Check out the [TechCrunch exclusive](https://docs.meilisearch.com/#:~:text=TechCrunch%20exclusive,opens%20new%20window) or learn more [on our blog](https://blog.meilisearch.com/meilisearch-series-a/).
+  New here? Check out the [Quick start guide](/learn/getting_started/quick_start.mdx) and learn how to launch a basic Meilisearch instance.
```
Comment (dichotommy) [inline home.mdx:13]: I feel pretty confident that this will fail since I don't think inserting Markdown into JSX is supported but let's try it as a test 😎

**Review by dichotommy — CHANGES_REQUESTED** [review]

File: home.mdx (hunk 4 above)
Comment (guimachiavelli) [reply home.mdx]: I'm pretty sure this works. Looking at files converted by basement, many use e.g. bold inside `<Capsule>`s:  https://meilisearch-staging.vercel.app/docs/learn/advanced/asynchronous_operations#task-queue (Note right above task queue, for example).

My understanding is that markdown is fine inside components, but fails inside props.

🟢 
```jsx
<Component>
  _Oxford comma_
</Component>
```

❌ 
```jsx
<Component propertyName="_Oxford comma_"/>
```

File: home.mdx (hunk 2 above)
Comment (guimachiavelli) [reply home.mdx:29]: I thought we were trying to avoid words like "easy" and "simple"?

Comment (dichotommy) [reply home.mdx:29]: We do avoid those words in a documentation context, but I don't think we ever decided to avoid them in a marketing context.

Comment (dichotommy) [reply home.mdx:29]: Either way, this suggestion is pretty subjective so I'm fine leaving it as is.

File: home.mdx (hunk 4 above)
Comment (dichotommy) [reply home.mdx]: Makes sense, I think you're right! Time to find out for sure 😎 🚀

**Review by dichotommy — APPROVED** [review]
Let's merge this and see how it looks (and what, if anything, it breaks)

**Comment by guimachiavelli** [comment]
bors merge

**Pull request merged**